generator/FactoryFile.php

```diff
@@ -83,6 +83,22 @@ public function generateDeclarationArguments(FactoryMethod $method)
         }
     }
 
+    public function generateReturnType(FactoryMethod $method): string
+    {
+        $call = $method->getCalls()[0];
+        if (!$call instanceof FactoryCall) {
+            throw new Exception('The first call in the FactoryMethod cannot be used to determine the return type. Method: '.$method->getName());
+        }
+
+        $returnType = $call->getMethod()->getReturnType();
+
+        if (!$returnType) {
```

Comment:
Where the `dump` function was defined and why it was removed?

Reply:
I temporarly added
https://symfony.com/doc/current/components/var_dumper.html
as a dev dependency, for debugging.

But forgot to remove it in this PR :)